### ISSUE TITLE
Crawling JSON

### ISSUE THREAD

**Issue by coprisanu** [Norconex/collector-http#601]
Hi,

       We need to crawl a JSON file and to split its content into smaller documents to be indexed in Elasticsearch. We have noticed there are already implementations like CVSSplitter, DOMSplitter or PDFsplitter, is there one for JSON?

Thank you

**Comment by essiembre** [comment]
No, there are currently none.  Good idea though.  I will mark as a feature request.  In the meantime, if you know your Java, you can implement your own solution by extending ``AbstractDocumentSplitter`` (feel free to share).
